configs/sites/tier1/blueback/packages.yaml

```diff
@@ -1,4 +1,22 @@
 packages:
+# Modification of common packages
+  # Use system zlib instead of spack-built zlib-ng
+  all:
+    providers:
+      zlib-api:: [zlib]
+  zlib-api:
+    buildable: False
+  zlib:
+    buildable: False
+    externals:
+    # For TDS tusk
+    - spec: zlib@1.2.11
+      prefix: /usr
+    # For TDS cole
```

Comment:
The blueback site config will be split into separate site configs for TDS cole, tusk in a follow-up PR.